### PR TITLE
Homepage first draft

### PR DESCRIPTION
**Pull request by jamari-morrison**
This PR contains the frist draft of the homepage (html+css). Would like to fix the color scheme to be more appealing and want to fill in actual data in the cards.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,19 +1,52 @@
...
+                <div class="topic-img-container"><img src="images/topic-3.png" alt="topic 3 image" ></div>
+                
+                <p>Topic description. Here, we can put some info about the topic, but this isn't a topic, it's a picture of c.c. from code geass  </p>
+                <a class="topic-btn" href="">LEARN MORE</a>
+            </div>
+        </div>
     </div>
-  </body>
-</html>
+
```
Comment (ElmerAdrianV) [inline portfolio/src/main/webapp/index.html:51]: LGTM!! All it's well, but maybe u can add a footer with about us that contains our names and the goal of the project :D

File: portfolio/src/main/webapp/home-styles.css [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+    left: 57%;
+    top: 3%;
+    position: relative;
+
+
+}
+.topic-btn:hover {
+	background:linear-gradient(to bottom, #dfdfdf 5%, #ededed 100%);
+	background-color:#dfdfdf;
+}
```
Comment (ElmerAdrianV) [inline portfolio/src/main/webapp/home-styles.css:126]: Info comment:D I will try to use this css file to styling the input-form for the page have style consistent

**Review by ElmerAdrianV — APPROVED** [review]
LGTM!! Nice job! Maybe can be helpful add in a comment one screenshot of the page to be more illustrative.

File: portfolio/src/main/webapp/index.html (hunk 1 above)
Comment (jamari-morrison) [reply portfolio/src/main/webapp/index.html:51]: yeah I think a footer is a great addition actually, we could also include any sort of resources we use in there as well

File: portfolio/src/main/webapp/home-styles.css (hunk 2 above)
Comment (jamari-morrison) [reply portfolio/src/main/webapp/home-styles.css:126]: Yeah, feel free to change it up however you'd like, I'm not exactly the best when it comes to colors and style and such

**Pull request merged**